packages/transport/src/car/codec.js

```diff
@@ -110,17 +110,16 @@ export const decode = async (bytes) => {
 
 /**
  * @param {Uint8Array} bytes
- * @param {{hasher?: API.MultihashHasher }} [options]
+ * @param {{hasher?: API.UCAN.MultihashHasher }} options
```

Comment:
Options should be optional

```suggestion
 * @param {{hasher?: API.UCAN.MultihashHasher }} [options]
```

Reply:
Turns out TS infers it as optional because default is set